src/config/graphical-session/gnome.md

```diff
@@ -0,0 +1,34 @@
+# GNOME
+
+## Pre-installation
+
+GNOME requires a display server such as [Xorg](./xorg.md) or a
+[Wayland](./wayland.md) compositor.
+
+Install the `elogind` and `adwaita-icon-theme` packages, ensure the `dbus`
+service is enabled, and reboot for the changes to take effect.
+
+## Installation
+
+Install the `gnome` package for a GNOME environment which includes GNOME
+applications.
+
+A minimal GNOME environment can be created by installing the `gnome-session` and
+`gdm` packages. Note, however, that such an minimal environment may not work
+well.
```

Comment:
I feel like this could be worded better as 'not all features may be present' or 'functional' or something?

Reply:
That's the sort of phrasing i was looking for! How about: "Note, however, that not all GNOME features may be present or functional"?

Reply:
We could mention `elogind` and `adwaita-icon-theme` here, instead. `gdm` seems to pull in elogind already, so we'd only need to mention adwaita. Do we need GDM for a basic GNOME session?

So the "minimal GNOME environment" would require `gnome-session`, `gdm` (or `elogind`) and `adwaita-icon-theme`.

Reply:
We should probably require gdm, it's what pulls in the GNOME window manager.